.readthedocs.yaml

```diff
@@ -0,0 +1,21 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-20.04
```

Comment:
Is there a reason not to use a more recent Ubuntu 22.04? Ubuntu 20.04 gets freely available updates only for two more years, and it's likely that readthetdocs will drop it then.

Reply:
This was the default from the [readthedocs documentation](https://docs.readthedocs.io/en/stable/config-file/v2.html#build-os), but it looks like `ubuntu-22.04` is also supported, so I'll bump it up to that.